docs/api/alerting/list_rule_types.asciidoc

```diff
@@ -4,7 +4,11 @@
 <titleabbrev>List rule types</titleabbrev>
 ++++
 
-Retrieve a list of alerting rule types.
+Retrieve a list of alerting rule types that the user is authorized to access.
+
+Each rule type includes a list of consumer features within which the user is authorized to perform either `read` or `all` operations on rules of that type. This can be useful to determine which rule types a user can read but not create or modify.
```

Comment:
```suggestion
Each rule type includes a list of consumer features. Users are authorized to perform either `read` or `all` operations on rules of that type. This helps determine which rule types users can read, but not create or modify.
```

Reply:
Thanks Gail... my concern with this wording is that it isn't clear from that rephrasing that the user is then only authorized to read/all these rule types when they are within that list of "consumer features".
If a feature isn't on the list, then the user is not allowed to read/modify rules of that type, if they are within the missing feature.

Does that make sense?

Reply:
Would this work? 
```suggestion
Each rule type includes a list of consumer features. Within these features, users are authorized to perform either `read` or `all` operations on rules of that type. This helps determine which rule types users can read, but not create or modify.
```

Reply:
I like it!

Reply:
Yay, thanks:)